repose-aggregator/src/docs/asciidoc/services/uri-redaction.adoc

```diff
@@ -0,0 +1,109 @@
+= URI Redaction service
+
+The URI Redaction service enables *Repose* to redact sensitive information contained in a URI.
+
+This is extremely useful for removing user tokens from API calls prior to creating OpenTracing spans.
+The URI Redaction service can be used by any service or filter, even custom ones, that need to redact any information from a URI during processing.
+
+== Configuration
+* *Default Configuration:* uri-redaction.cfg.xml
+* *Released:* v8..?.?.?
```

Comment:
Since we think the next release is `8.8.4.0` (and in sprint), do we want to go ahead and populate that here?

Reply:
Sure. Updated.